test/test_sparse_semi_structured.py

```diff
@@ -341,6 +341,44 @@ def test_mlp(self, device, dense_input_shape, backend):
 
         assert torch.allclose(dense_result, sparse_result, rtol=1e-3, atol=1e-3)
 
+    @parametrize("backend", SEMI_STRUCTURED_SUPPORTED_BACKENDS)
+    @parametrize("dense_input_shape", [(1, 128), (64, 128), (128, 128), (64, 128, 128)])
+    def test_mlp_contiguous_relu_compile(self, dense_input_shape, backend, device):
```

Comment:
oh for the failing trunk CI - you probably need to mark this test as skipped when running in windows CI:

```
from torch.testing._internal.common_utils import IS_WINDOWS
 @unittest.skipIf(IS_WINDOWS, "torch.compile doesn't work with windows")
```